hack/components/bump-multus.sh

```diff
@@ -106,11 +106,47 @@ rm -rf data/multus/*
 cp ${MULTUS_PATH}/config/cnao/000-ns.yaml data/multus/
 cp ${MULTUS_PATH}/config/cnao/001-multus.yaml data/multus/
 
-echo 'Get multus image name and update it under CNAO'
+echo 'Build multus image, push it to quay.io and update it under CNAO'
 MULTUS_TAG=$(git-utils::get_component_tag ${MULTUS_PATH})
-MULTUS_IMAGE=nfvpe/multus
+MULTUS_IMAGE=quay.io/kubevirt/cluster-network-addon-multus
 MULTUS_IMAGE_TAGGED=${MULTUS_IMAGE}:${MULTUS_TAG}
-MULTUS_IMAGE_DIGEST="$(docker-utils::get_image_digest "${MULTUS_IMAGE_TAGGED}" "${MULTUS_IMAGE}")"
+(
+    cd ${MULTUS_PATH}
+    cat <<EOF > Dockerfile
+
+# This dockerfile is specific to building Multus for OpenShift
```

Comment:
But this image will be never used on OpenShift. Could we just drop the misleading comment?